astroquery/alma/core.py

```diff
@@ -60,13 +73,10 @@ def query_object_async(self, object_name, cache=True, public=True,
         kwargs : dict
             Passed to `query_async`
         """
-
-        if payload is None:
-            payload = {}
-        payload.update({'source_name_resolver': object_name, })
-
-        return self.query_async(payload, cache=cache, public=public,
-                                science=science, **kwargs)
+        obj_coord = coord.SkyCoord.from_name(object_name)
+        return self.query_region_async(obj_coord, radius=10 * u.arcmin,
```

Comment:
Is this how the ALMA archive is presently doing their queries - i.e., resolving the name, then searching in a 10 arcminute radius?  Seems sketchy...